server/hotreloadmiddleware/process-update.js

```diff
@@ -33,14 +33,8 @@ module.exports = function(hash, moduleMap, options) {
       if (err) return handleError(err);
 
       if(!updatedModules) {
-        if (options.warn) {
-          console.warn("[HMR] Cannot find update (Full reload needed)");
-          console.warn("[HMR] (Probably because of restarting the server)");
-        }
-        performReload();
```

Comment:
And everything still reloads as appropriate?

Reply:
Kris’ comment seems to suggest that it doesn’t. The whole point of hot reload was to update CSS and HTML on the fly. 

Config changes (component.json) require restart so they are outside the hot reload umbrella, but it would be nice if we could somehow used dynamic config to change it under the hood. Not for this review, just wishful thinking.

Reply:
Apparently they are still rebuilt, but developer has to hit CTRL+R to reload and see them (or whack a change in the JS file)... Webpack only watches the JS (other processes watch the HTML and CSS). However I still think there's a hack somewhere along the line that would make this work (like having the other watch function touch the JS file when they are rebuilt).

Reply:
Again, this is the whole point of the hot reload. Webpack will build the files, but hot reload was added to automatically reload the page to smooth development. If that doesn't work, then this needs to go back to the drawing board.

Reply:
Agree... If possible CSS and HTML changes should cause a component reload - however if that can't be achieved without hotreload hangs then we'll need to improve docs so that devs at least know they need to hit a manual reload.